src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Controller.cs

```diff
@@ -69,6 +69,16 @@ internal partial class Controller :
             _completionBroker = completionBroker;
         }
 
+        public event EventHandler<Model> ModelUpdated;
```

Comment:
nit: consider naming `AfterModelUpdated`

Reply:
The current name [follows naming guidelines](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1713)